apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -34,11 +34,12 @@ public class NPIOrgLookup {
    * Factory method for creating a {@link NPIOrgLookup } for testing that includes the fake org
    * name.
    *
+   * @throws IOException if there is an issue reading file
    * @return the {@link NPIOrgLookup }
    */
   public static NPIOrgLookup createNpiOrgLookupForTesting() throws IOException {
```

Comment:
We should be able to get rid of createNpiOrgLookupForTesting, createNpiOrgLookupForProduction, and the empty test constructor now; just use the single constructor that takes the file to load. Prod will load the prod file, tests will load the test file.